### ISSUE TITLE
Runtime Error : module 'lm_pnp' has no attribute 'compute_lm_pnp'`

### ISSUE THREAD

**Issue by mhmtsarigul** [sfu-gruvi-3dv/sanet_relocal_demo#10]
I have built lm_pnp and pointnet as in "https://github.com/sfu-gruvi-3dv/sanet_relocal_demo". But when I try to run example_7scenes i got an error in "Query a single frame and visualize the scene coordinates". 

`---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-19-448705329560> in <module>()
    114                                                 query_K,
    115                                                 pnp_x_2d,
--> 116                                                 repro_thres=repro_thres)
    117 
    118 print('rot err: %f, translation err: %f' % (R_acc, t_acc))

/blabla/sanet_relocal_demo/reloc_pipeline/util_func.py in compute_pose_lm_pnp(gt_Tcws, query_X_w, rand_R, scene_center, query_K, pnp_x_2d, repro_thres)
    348 
    349     # run Ransac PnP
--> 350     lm_pnp_pose_vec, inlier_map = lm_pnp.compute_lm_pnp(pnp_x_2d, query_X_3d_w, query_K, repro_thres, 128, 100)
    351     R_res, _ = cv2.Rodrigues(lm_pnp_pose_vec[:3])
    352     lm_pnp_pose = np.eye(4, dtype=np.float32)

AttributeError: module 'lm_pnp' has no attribute 'compute_lm_pnp'`

When I check "lm_pnp.py" I can see there is just 2 line of codes.
` "import sys 
sys.path.append('')"`

What am I doing wrong? 

Thanks for sharing your great work.

**Comment by corsy** [comment]
Can you add the path that include the lm_pnp_*.so into PYTHONPATH environment variable and try if you can `import lm_pnp` using ipython?

**Comment by mhmtsarigul** [comment]
I have "lm_pnp.cpython-36m-x86_64-linux-gnu.so" under /libs/lm_pnp/build folder and i already added both /libs/lm_pnp and /libs/lm_pnp/build to pythonpath by using sys.path.append. But still `AttributeError: module 'lm_pnp' has no attribute 'compute_lm_pnp'`. When I remove lm_pnp.py, I got the error there is no module named lm_pnp.py. I am working on colab, I don't know if it creates any problem, but all paths seem correct. 

syspath : ['', '/content', '/env/python', '/usr/lib/python37.zip', '/usr/lib/python3.7', '/usr/lib/python3.7/lib-dynload', '/usr/local/lib/python3.7/dist-packages', '/usr/lib/python3/dist-packages', '/usr/local/lib/python3.7/dist-packages/IPython/extensions', '/root/.ipython', '/content/drive/MyDrive/foldershere/sanet_relocal_demo/libs/lm_pnp/build', './libs/lm_pnp', '', './libs/pycbf_filter', '', '/content/drive/MyDrive/foldershere/sanet_relocal_demo/libs/lm_pnp']

**Comment by mhmtsarigul** [comment]
I figured out the problem. Colab does not take python 3.7 automatically for CMake operation but still does not show any error in compiling. After I add set(PYTHON_EXECUTABLE /usr/bin/python3.7) to all cmakelist files, I was able to add .so files to my notebooks. Still, I needed to conda install the pybind11 for cmake first, after pip install pybind for usage without any error. 

Thanks for sharing your work.
